### ISSUE TITLE
Question Evil Portal

### ISSUE THREAD

**Issue by ghost** [Shlucus/FlipperZero-GooglePortal#2]
Hello, do you aim to make other evil portals?
THANKS

**Comment by Shlucus** [comment]
Yes! Google has actually made a new UI for their login pages. I'm likely to make another one based on that. And besides Google, I'll be likely to create some based on popular social media platforms (most likely those that lack 2FA) and maybe even companies/places exclusive to my current city.

**Comment by Shlucus** [comment]
> Hello, do you aim to make other evil portals? THANKS

Hey softm5, in the next month, I'm planning on adding to the repo Google's new Sign-In UI. So stay posted!
